### PR TITLE
add scheduler

### PR DESCRIPTION
**Pull request by AlvinRadeka**
# Changelog

- Add scheduler package

## The changes contain

- [x] new features
- [ ] bugfix
- [ ] code improvements
- [ ] breaking changes

### CODE REVIEW

File: scheduler/scheduler.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	"github.com/downsized-devs/sdk-go/logger"
+	"github.com/go-co-op/gocron/v2"
+)
+
+const (
+	Duration       = "duration"
+	RandomDuration = "random-duration"
+	Daily          = "daily"
+	Weekly         = "weekly"
+	Monthly        = "monthly"
```
Comment (BamWidyat) [inline scheduler/scheduler.go:17]: tambahin cron?

Comment (AlvinRadeka) [reply scheduler/scheduler.go:17]: Nyusul ya, yg cron males bikinnya wkwkwk

Comment (BamWidyat) [reply scheduler/scheduler.go:17]: Wkwkwk

**Comment by BamWidyat** [comment]
Testnya fail tuh

**Review by BamWidyat — APPROVED** [review]

**Pull request merged**